Our `Nightly` CD is failing in `decrypt files` job.

**Describe the bug**
Our `Nightly` CD is failing. See https://github.com/kiwix/kiwix-android/actions/runs/20766026269/job/59632201516

**Expected behavior**
It should pass.

**Environment**
- Version of Kiwix Android: 4.0.0

**Logs**
```kotlin
Run echo "$KEYSTORE" | base64 -d > ***-android.keystore
  echo "$KEYSTORE" | base64 -d > ***-android.keystore
  echo "$KIWIX_FILE_UPLOAD_SSH_KEY" | base64 -d > $KIWIX_FILE_UPLOAD_SSH_KEY_PATH
  chmod 600 $KIWIX_FILE_UPLOAD_SSH_KEY_PATH
  shell: /usr/bin/bash -e {0}
  env:
    JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.17-10/x64
    JAVA_HOME_17_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.17-10/x64
    KEYSTORE: ***
    KIWIX_FILE_UPLOAD_SSH_KEY: ***
    KIWIX_FILE_UPLOAD_SSH_KEY_PATH: ***_file_upload_ssh_key
base64: invalid input
Error: Process completed with exit code 1.
```

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -32,11 +32,12 @@ jobs:
         env:
           KEYSTORE: ${{ secrets.keystore }}
           KIWIX_FILE_UPLOAD_SSH_KEY: ${{ secrets.KIWIX_FILE_UPLOAD_SSH_KEY }}
-          KIWIX_FILE_UPLOAD_SSH_KEY_PATH: kiwix_file_upload_ssh_key
         run: |
           echo "$KEYSTORE" | base64 -d > kiwix-android.keystore
-          echo "$KIWIX_FILE_UPLOAD_SSH_KEY" | base64 -d > $KIWIX_FILE_UPLOAD_SSH_KEY_PATH
-          chmod 600 $KIWIX_FILE_UPLOAD_SSH_KEY_PATH
+          KEY_PATH=kiwix_file_upload_ssh_key
+          echo "$KIWIX_FILE_UPLOAD_SSH_KEY" > $KEY_PATH
+          chmod 600 $KEY_PATH
+          echo "KIWIX_FILE_UPLOAD_SSH_KEY_PATH=$KEY_PATH" >> $GITHUB_ENV
 
       - name: build debug
         env:
@@ -55,4 +56,4 @@ jobs:
         run: |
           mkdir $DATE
           cp $UNIVERSAL_DEBUG_APK $DATE
-          scp -P 30022 -vrp -i $KIWIX_FILE_UPLOAD_SSH_KEY_PATH -o StrictHostKeyChecking=no $DATE ci@master.download.kiwix.org:/data/download/nightly/
+          scp -P 30022 -vrp -i "$KIWIX_FILE_UPLOAD_SSH_KEY_PATH" -o StrictHostKeyChecking=no $DATE ci@master.download.kiwix.org:/data/download/nightly/

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -38,11 +38,12 @@ jobs:
         env:
           KEYSTORE: ${{ secrets.keystore }}
           KIWIX_FILE_UPLOAD_SSH_KEY: ${{ secrets.KIWIX_FILE_UPLOAD_SSH_KEY }}
-          KIWIX_FILE_UPLOAD_SSH_KEY_PATH: kiwix_file_upload_ssh_key
         run: |
           echo "$KEYSTORE" | base64 -d > kiwix-android.keystore
-          echo "$KIWIX_FILE_UPLOAD_SSH_KEY" | base64 -d > $KIWIX_FILE_UPLOAD_SSH_KEY_PATH
-          chmod 600 $KIWIX_FILE_UPLOAD_SSH_KEY_PATH
+          KEY_PATH=kiwix_file_upload_ssh_key
+          echo "$KIWIX_FILE_UPLOAD_SSH_KEY" > $KEY_PATH
+          chmod 600 $KEY_PATH
+          echo "KIWIX_FILE_UPLOAD_SSH_KEY_PATH=$KEY_PATH" >> $GITHUB_ENV
 
       - name: Publish APK to download.kiwix.org
         env:
@@ -56,14 +57,14 @@ jobs:
         run: |
           ./gradlew assembleStandalone
           cp ${UNIVERSAL_RELEASE_APK} ${ARCHIVE_NAME}
-          scp -P 30022 -vrp -i $KIWIX_FILE_UPLOAD_SSH_KEY_PATH -o StrictHostKeyChecking=no "$ARCHIVE_NAME" ci@master.download.kiwix.org:/data/download/release/kiwix-android/
+          scp -P 30022 -vrp -i "$KIWIX_FILE_UPLOAD_SSH_KEY_PATH" -o StrictHostKeyChecking=no "$ARCHIVE_NAME" ci@master.download.kiwix.org:/data/download/release/kiwix-android/
 
 #      This is temporary, once we will publish 3.7.0 then we will uncommented this code.
 #      # This is necessary for F-Droid
 #      - name: Publish "versionInfo" to download.kiwix.org
 #        run: |
 #          ./gradlew generateVersionCodeAndName
-#          scp -P 30022 -vrp -i $KIWIX_FILE_UPLOAD_SSH_KEY_PATH -o StrictHostKeyChecking=no VERSION_INFO ci@master.download.kiwix.org:/data/download/release/kiwix-android/
+#          scp -P 30022 -vrp -i "$KIWIX_FILE_UPLOAD_SSH_KEY_PATH" -o StrictHostKeyChecking=no VERSION_INFO ci@master.download.kiwix.org:/data/download/release/kiwix-android/
 
       - name: Upload APKs to Release
         uses: ncipollo/release-action@v1

